### PR TITLE
AnimationTree sliders fix

### LINKED ISSUE #8093: AnimationTreePlayer: animation blend/mix slider is not working in the editor
**Operating system or device - Godot version:**
In Godot 2.1 everything is working fine. In Godot 2.1.1 bug is present already.

**Issue description:**
Blending is working properly in Godot 2.1: 
blend_good

Since Godot 2.1.1 it's not working anymore:
blend_bad

Also this is an editor bug, during runtime everything seems to work fine, so it might be possible it's just something with ui of blend/mix node.

**Steps to reproduce:**
1. Download Platformer 3D demo
2. Open "Player.scn"
3. Select AnimationTreePlayer node
4. Try to change animation blending in "gun" node of Animation Tree Player

**Link to minimal example project:**

### PR DESCRIPTION
**Pull request by kubecz3k**
This commit will ensure that `step property of concrete `Range` widgets which are used by `AnimationTreePlayer` will be set to 0.01 (one percent).

However I'm wondering why default values for step is 1.0 (100%), I don't think that makes a lot of sense. Also default range is 0-100 (0%-10 000%). 
We actually have somewhere in Godot UI such slider? Mean the one that uses those defaults? 
Current default in range: https://github.com/godotengine/godot/blob/master/scene/gui/range.cpp#L275

This pr will close #8093

### CODE REVIEW

**Comment by kubecz3k** [comment]
Oh my, by accident I had pushed two commits, will fix in a minute

File: editor/plugins/animation_tree_editor_plugin.cpp [hunk 1]
```diff
@@ -419,6 +423,7 @@ void AnimationTreeEditor::_popup_edit_dialog() {
 				edit_label[1]->show();
 				edit_scroll[1]->set_min(0);
 				edit_scroll[1]->set_max(1);
+				edit_scroll[0]->set_step(0.01);
```
Comment (Noshyaar) [inline editor/plugins/animation_tree_editor_plugin.cpp]: I think this should be `[1]`?

Comment (kubecz3k) [inline editor/plugins/animation_tree_editor_plugin.cpp]: good catch!

Comment (kubecz3k) [inline editor/plugins/animation_tree_editor_plugin.cpp]: I'm sorry for fuss, but I closed this pr and opened a new one here: https://github.com/godotengine/godot/pull/9749
This pr would push 2 commits into master and Akien would not be happy about that...

**Pull request closed without merging**